src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -36,6 +40,7 @@
 
     private String name;
     private String url;
+    private String urlCredentialsId;
```

Comment:
then add documentation what is the purpose of having first and the second url

Reply:
Added a comment here in the code